sqrl-planner/src/main/java/com/datasqrl/engine/database/QueryEngine.java

```diff
@@ -13,9 +20,14 @@
  */
 public interface QueryEngine extends ExecutionEngine {
 
-
-  default Map<IdentifiedQuery, QueryTemplate> updateQueries(
-      ConnectorFactoryFactory connectorFactory, EngineConfig connectorConfig, Map<IdentifiedQuery, QueryTemplate> queries) {
-    return queries;
+  @Override
+  default DatabasePhysicalPlan plan(StagePlan plan, List<StageSink> inputs,
```

Comment:
This 'updateQueries' is a bit of an antipattern for the engine planning. We can safely just delegate the queries to the engine and allow it to part of its own physical plan, which lets this be removed entirely. This is my fault and I'll correct it on top of this PR.